tick.go

```diff
@@ -70,36 +73,103 @@ func generateContinuousTicks(r render.Renderer, ra sequence.Range, isVertical bo
 	domainRemainder := domain - (tickSize * 2)
 	intermediateTickCount := int(math.Floor(float64(domainRemainder) / float64(tickSize)))
 
-	rangeDelta := math.Abs(max - min)
-	tickStep := rangeDelta / float64(intermediateTickCount)
+	formatterName := runtime.FuncForPC(reflect.ValueOf(vf).Pointer()).Name()
+	if strings.HasSuffix(formatterName, "FloatValueFormatter") {
```

Comment:
is it possible to check what `vf` type is like a type switch or something? Looks a bit hacky to do this kind of string search

Reply:
I've replace this check by checking the first tick label instead, since I couldn't use switch to check the function type. If the first tick label is a float number then use the nice ticks calculation.